terraform/gcp-fxci-production-level1-workers/_tc_credentials.tf

```diff
@@ -0,0 +1,10 @@
+# required for worker-manager to be able to do anything in GCP
+resource "google_project_iam_member" "worker-manager-compute-admin" {
+  project = "fxci-production-level1-workers"
+  role    = "roles/compute.admin"
+  member  = "serviceAccount:taskcluster-worker-manager@fxci-production-level1-workers.iam.gserviceaccount.com"
+}
+
+# TODO:
+# - taskcluster-worker@fxci-production-level1-workers.iam.gserviceaccount.com
+# - deploy-prod@fxci-production-level1-workers.iam.gserviceaccount.com
```

Comment:
Yes, eventually.

Since this is the one role that we needed to add back to get things working and we're focused on getting this code to a safe state, let's punt for now on the other roles.

`deploy-prod` currently has the `Compute Admin` role, but I'm not sure how. I thought the authoritative `google_project_iam_binding` would remove all other users with `Compute Admin`, but it's there and I didn't add it back after the incident.

Reply:
Ah, sorry... I was thinking deploy-prod was an older TC-related account, not the new one you just made. I will add.

Reply:
OK, so `Compute Admin` was already being granted to `deploy-prod` in the _github_oidc.tf file.

I think we're good.